### PR TITLE
Add fdb flush lua script v2 with switch id support

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

**Comment by lguohan** [comment]
can you describe the motivation if this change?  why we do not need it before?

**Comment by kcudnik** [comment]
if you  flush fdb on one switch when multiple switches are present in syncd currently it will flush fdb entries from all switches, this change will add support per switch, then previous version can be removed

File: tests/Makefile.am [hunk 1]
```diff
@@ -27,6 +27,7 @@ tests_SOURCES = redis_ut.cpp                \
                 selectable_priority.cpp       \
                 warm_restart_ut.cpp         \
                 redis_multi_db_ut.cpp       \
+				fdb_flush.cpp				\
```
Comment (qiluo-msft) [inline tests/Makefile.am]: >				 [](start = 0, length = 4)

tab->spaces

File: tests/fdb_flush.cpp [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+{
+    DBConnector db("TEST_DB", 0, true);
+
+    ProducerTable producer(&db, tableName);
+    ConsumerTable consumer(&db, tableName);
+
+    // "SAI_OBJECT_TYPE_FDB_ENTRY:{\"bvid\":\"oid:0x26000000000545\",\"mac\":\"4A:A8:7B:A2:37:1A\",\"switch_id\":\"oid:0x21000000000000\"}"
+    // "SAI_FDB_ENTRY_ATTR_TYPE" "SAI_FDB_ENTRY_TYPE_DYNAMIC"
+    // "SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID" "oid:0x3a000000000567"
+    // "SAI_FDB_ENTRY_ATTR_PACKET_ACTION" "SAI_PACKET_ACTION_FORWARD"
```
Comment (qiluo-msft) [inline tests/fdb_flush.cpp]: remove dead code

File: tests/fdb_flush.cpp [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+    mac(1,1);
+    mac(2,0);
+    mac(3,1);
+    mac(4,0);
+
+    //insert(0x21000000000000, 0x26000000000001, 0x3a000000000001, 1, true);
+    //insert(0x21000000000000, 0x26000000000002, 0x3a000000000002, 2, false);
+    //insert(0x121000000000001, 0x126000000000003, 0x13a000000000003,  3, true);
+    //insert(0x121000000000001, 0x126000000000004, 0x13a000000000004,  4, false);
```
Comment (qiluo-msft) [inline tests/fdb_flush.cpp]: remove dead code

**Pull request merged**